### PR TITLE
Forgot to push this, sorry

### PR DESCRIPTION
**Pull request by agustif**
It's needed to show the todo's, I deleted it wrongly thinking it was part of my discarded sidebar-summary code!

### CODE REVIEW

File: server/services/cli.ts [hunk 1]
```diff
@@ -1 +0,0 @@
-export const root = process.env.ROOT || process.cwd();
```
Comment (Raathigesh) [inline server/services/cli.ts]: I don't think we want to delete this :)

File: ui/test-file/summary/index.tsx [hunk 2]
```diff
@@ -148,6 +152,9 @@ export default function FileSummary({
           <Info color="primary">
             <Code size={14} /> <InfoLabel>{testCount} Tests</InfoLabel>
           </Info>
+          <Info color="secondary">
+          <Circle size={14} />{" "} <InfoLabel>{todoCount} Todos</InfoLabel>
```
Comment (Raathigesh) [inline ui/test-file/summary/index.tsx]: Also is this space necessary?

File: server/services/cli.ts (hunk 1 above)
Comment (agustif) [reply server/services/cli.ts]: Nope, messing it up again, I wanted to delee my cli as I was using a custom string sorry too late to PR I guess

File: ui/test-file/summary/index.tsx (hunk 2 above)
Comment (agustif) [reply ui/test-file/summary/index.tsx]: Don't think so, just copied from the other example as is

**Comment by agustif** [comment]
So sorry for this mess, ide/ auto-indentation was making a mess of a PR and I'm trying to save it with as minimum delete/inserts as possible

File: ui/test-file/summary/index.tsx (hunk 2 above)
Comment (agustif) [reply ui/test-file/summary/index.tsx]: deleted on commit ca7f016

**Comment by agustif** [comment]
I've gone ahead and opened a new clean PR, closing this one

**Pull request closed without merging**